tests/tests_integration/test_session/test_graph_flow.py

```diff
@@ -99,10 +103,53 @@ def test_dexpi_to_dms(self, cognite_client: CogniteClient, data_regression: Data
             # In progress, not yet supported.
             dms_rules = neat._state.data_model.last_verified_dms_rules[1]
             store = neat._state.instances.store
-            instances = [
-                self._standardize_instance(instance)
-                for instance in DMSLoader.from_rules(dms_rules, store, "sp_instance_space").load()
-            ]
+            instances = list(DMSLoader.from_rules(dms_rules, store, "sp_instance_space").load())
+
+            nodes = [instance for instance in instances if isinstance(instance, NodeApply)]
+            edges = [instance for instance in instances if isinstance(instance, EdgeApply)]
+        else:
+            instances = []
+
+        remove_linking_in_rules(neat._state.data_model.last_verified_dms_rules[1])
+        rules_str = neat.to.yaml(format="neat")
+
+        rules_dict = yaml.safe_load(rules_str)
+        data_regression.check(
+            {
+                "rules": rules_dict,
+                "instances": [],
+            }
+        )
+
```

Comment:
we did not test if and how many edges are we creating, we only look at raw no of instances and missed the fact that we were not producing edges for some time